### PR TITLE
ODF 4.12 release

### PR DESCRIPTION
**Pull request by gayathrimenath-ibm**

### CODE REVIEW

**Review by akgunjal — APPROVED** [review]
lgtm

**Review by akgunjal — APPROVED** [review]
lgtm

File: config-templates/redhat/odf-local/4.12/deployment.yaml [hunk 1]
```diff
@@ -0,0 +1,483 @@
+apiVersion: v1
+kind: List
+metadata:
+  name: ibm-openshift-container-storage
+  namespace: kube-system
+  annotations:
+    revision: 2
```
Comment (mssachan) [inline config-templates/redhat/odf-local/4.12/deployment.yaml]: change revision to `1`

File: config-templates/redhat/odf-local/4.12/metadata.json [hunk 2]
```diff
@@ -0,0 +1,77 @@
+{
+  "name": "4.12",
+  "revision": "2",
```
Comment (mssachan) [inline config-templates/redhat/odf-local/4.12/metadata.json]: change `revision` to `1`

File: config-templates/redhat/odf-local/4.12/storage-class.yaml [hunk 3]
```diff
@@ -0,0 +1,139 @@
...
+kind: List
+items:
+  - apiVersion: storage.k8s.io/v1
+    kind: StorageClass
+    metadata:
+      name: sat-ocs-cephrbd-gold
+      labels:
+        app.kubernetes.io/name: "redhat-odf-local"
+        app.kubernetes.io/version: "v4.12.0"
+        app.kubernetes.io/revision: "2"
```
Comment (mssachan) [inline config-templates/redhat/odf-local/4.12/storage-class.yaml]: change `revision` to `1` at all places in this file

File: config-templates/redhat/odf-remote/4.12/deployment.yaml [hunk 4]
```diff
@@ -0,0 +1,479 @@
+apiVersion: v1
+kind: List
+metadata:
+  name: ibm-openshift-container-storage
+  namespace: kube-system
+  annotations:
+    revision: 2
```
Comment (mssachan) [inline config-templates/redhat/odf-remote/4.12/deployment.yaml]: change `revision` to `1`

File: config-templates/redhat/odf-remote/4.12/metadata.json [hunk 5]
```diff
@@ -0,0 +1,77 @@
+{
+  "name": "4.12",
+  "revision": "2",
```
Comment (mssachan) [inline config-templates/redhat/odf-remote/4.12/metadata.json]: change `revision` to `1`

File: config-templates/redhat/odf-local/changelog.json [hunk 6]
```diff
@@ -1,93 +1,116 @@
 [
     {
-        "4.11": {
+        "4.12": {
```
Comment (mssachan) [inline config-templates/redhat/odf-local/changelog.json:3]: "4.12": {
            "status": "supported",
            "latest-revision": {
                "revision": "1",
                "release-date": "04 April 2023",
                "cves": "",
                "golang": "1.19.7",
                "ubi": "8.7-1085.1679482090",
                "new-features": [
                      "Initial release"
                ],
                "fixes": []
            }
        }

File: config-templates/redhat/odf-remote/changelog.json [hunk 7]
```diff
@@ -1,93 +1,116 @@
 [
     {
-        "4.11": {
+        "4.12": {
```
Comment (mssachan) [inline config-templates/redhat/odf-remote/changelog.json:3]:
```
 "4.12": {
        "status": "supported",
        "latest-revision": {
            "revision": "1",
            "release-date": "04 April 2023",
            "cves": "",
            "golang": "1.19.7",
            "ubi": "8.7-1085.1679482090",
            "new-features": [
                  "Initial release"
            ],
            "fixes": []
        }
    }
```

**Review by mssachan — CHANGES_REQUESTED** [review]

**Review by mssachan — APPROVED** [review]
LGTM

**Pull request merged**